action/deployment_add.go

```diff
@@ -66,6 +66,12 @@ var DeploymentAdd = &cli.Command{
 			Usage:   "Provide the task for the deployment",
 			Value:   "deploy:vela",
 		},
+		&cli.StringSliceFlag{
+			EnvVars: []string{"VELA_PARAMETER", "DEPLOYMENT_PARAMETER"},
```

Comment:
Should these be plural since they are set once in list format?

```suggestion
			EnvVars: []string{"VELA_PARAMETERS", "DEPLOYMENT_PARAMETERS"},
```

I think it's acceptable to leave the name of the flag as `parameter` because if you were to provide multiple parameters, you have to provide the flag multiple times:

```sh
vela add deployment --org MyOrg --repo MyRepo --parameter 'key=value' --parameter 'foo=bar'
```

But if you were to set these as an env var, then it would have to be one env var provided in a list:

```sh
VELA_PARAMETERS='key=value,foo=bar' vela add deployment --org MyOrg --repo MyRepo
```

Reply:
Updated in the latest commit.